CuitService/Security/SecurityServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,49 @@
+using CuitService.Security;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Linq;
+using System.Security.Cryptography;
+
+namespace Microsoft.Extensions.DependencyInjection
```

Comment:
Well, it is obviously not "invalid" because it works.

Maybe it could be _inappropriate_, _wrong_ or _ugly_, but in my opinion, it is not because I am following framework conventions:

> Each `services.Add{SERVICE_NAME}` extension method adds (and potentially configures) services. For example, `services.AddMvc()` adds the services Razor Pages and MVC require. We recommended that apps follow this convention. **Place extension methods in the Microsoft.Extensions.DependencyInjection namespace to encapsulate groups of service registrations**.
> 
> from [Overview of dependency injection](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-3.1#overview-of-dependency-injection)